boards/arm/sparkfun_pro_mirco_rp2040/sparkfun_pro_micro_rp2040.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: raspberrypi_pico
+name: RaspberryPi-Pico
+type: mcu
+arch: arm
+flash: 2048
```

Comment:
Isn't it 16MB?

Reply:
Good catch. Fixed. Also had a c&p issue w/ the name and identifier fields that I fixed.